### PR TITLE
Collect Docker and docker-storage-setup configuration files

### PR DESCRIPTION
**Pull request by jahkeup**
This collects the configuration used for setting up docker and docker storage setup.

**WIP**: I have yet to test this on a debian or ubuntu host.

### CODE REVIEW

File: ecs-logs-collector.sh [hunk 1]
```diff
@@ -362,6 +363,41 @@ get_docker_logs() {
   ok
 }
 
+get_docker_config() {
+  try "collect system's Docker configuration"
+  dstdir="${info_system}/docker_config"
+  mkdir -p "${dstdir}"
+
+  # Find and collect system's supporting configuraton files.
+  for conf in /etc/sysconfig/docker /etc/sysconfig/docker-storage-setup /etc/default/docker; do
```
Comment (jhaynes) [inline ecs-logs-collector.sh:372]: We should grab `/etc/docker-storage` too.

File: ecs-logs-collector.sh [hunk 2]
```diff
@@ -362,6 +363,41 @@ get_docker_logs() {
...
+    if [ -f "${conf}" ]; then
+      install -D "${conf}" "${dstdir}${conf}"
+    fi
+  done
+
+  # Collect the docker.service unit if the system is running systemd.
+  if command -v systemctl > /dev/null; then
```
Comment (jhaynes) [inline ecs-logs-collector.sh:379]: Worth "collecting" info about how docker was started in other init systems?

**Review by jhaynes — APPROVED** [review]
LGTM, just potential things to collect also.

File: ecs-logs-collector.sh (hunk 2 above)
Comment (zmrow) [reply ecs-logs-collector.sh:379]: I agree, let's open a separate issue on this rather than delay this push

File: ecs-logs-collector.sh [hunk 3]
```diff
@@ -362,6 +363,41 @@ get_docker_logs() {
   ok
 }
 
+get_docker_config() {
+  try "collect system's Docker configuration"
+  dstdir="${info_system}/docker_config"
```
Comment (zmrow) [inline ecs-logs-collector.sh:368]: Nitpick: I realize that this convention doesn't exist in this repo, but I'd prefer `dstdir` vars inside of functions be declared as `local`. 

```
local dstdir="..."
```

This is truly a nitpick and probably doesn't need to be handled in this PR.

**Review by zmrow — APPROVED** [review]
Just a few comments.

I say ship it and we'll open separate issues if needed to address jhaynes@ and my comments

**Comment by jahkeup** [comment]
Closing in favor of updated and tested PR: #56

**Pull request closed without merging**